oxidation/libparsec/crates/platform_storage/src/error.rs

```diff
@@ -0,0 +1,59 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use libparsec_crypto::CryptoError;
+use libparsec_types::{uuid::Uuid, RegexError};
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
Can we keep naming the error ?

Reply:
Can you develop what you're meaning ? I'm not sure to understand ?

Reply:
Give a name to the error, like FSError

Reply:
done